examples/sla_policies.tf

```diff
@@ -6,9 +6,9 @@
 
 resource "zendesk_sla_policy" "incidents_sla_policy" {
   title  = "Incidents"
-  active = true
 
   all {
+    # TODO: this is failing on the ACC test.
```

Comment:
@nukosuke  after looking into this a bit more I think the problem here is that zendesk is translating the input here:


It is taking this:
```
  all {
    # TODO: this is failing on the ACC test.
    field    = "type"
    operator = "is"
    value    = "incident"
  }
```

and turning it into something like this

```
  all {
    # TODO: this is failing on the ACC test.
    field    = "ticket_type_id"
    operator = "is"
    value    = 2
  }
```

but that said I am not exactly sure of the best way to solve this. Seems like we might have to figure out how to translate the response from zendesk back into the resource listed in the terraform file. 

Curious if you have thoughts.


See: https://developer.zendesk.com/rest_api/docs/support/views#additional-sla-policies-conditions